### PR TITLE
update lsdc server env

### PR DESCRIPTION
**Pull request by JunAishima**
* update version number
 * while there are no changes in the dependency list here,
   the dependencies themselves have been significantly
   updated so a new environment is required

### CODE REVIEW

File: configs/lsdc-server.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 docker_image: "quay.io/condaforge/linux-anvil-cos7-x86_64:latest"
```
Comment (mrakitin) [inline configs/lsdc-server.yml:1]:
```suggestion
docker_image: "nsls2/linux-anvil-cos7-x86_64:latest"
```

**Review by mrakitin — COMMENTED** [review]
Any other changes here? Any updates for `lsdc-server-env.yml` to enforce the minimum pins of `conftrak`, `analysisstore`, `amostra`?

File: configs/lsdc-server.yml (hunk 1 above)
Comment (JunAishima) [reply configs/lsdc-server.yml]: done!

**Comment by JunAishima** [comment]
> Any other changes here? Any updates for `lsdc-server-env.yml` to enforce the minimum pins of `conftrak`, `analysisstore`, `amostra`?

You are right - the tornado changes are pretty big - just added the current versions as lower limits, and added the "<1" for amostra as well

**Review by mrakitin — APPROVED** [review]

**Pull request merged**